Makefile

```diff
@@ -41,3 +41,60 @@ javascript/dist/main.css: javascript/dist/main.js
 
 javascript/dist/main.js: javascript/package.json $(javascript_source)
 	cd javascript && npm install-test && npm run build
+
+RELEASED_VERSION = $(shell changelog latest)
+JAVA_VERSION = $(shell cd java && make version)
+RUBY_VERSION = $(shell cd ruby && make version)
+JAVASCRIPT_VERSION = $(shell cd javascript && make version)
+CURRENT_BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
+
+language-versions-check:
```

Comment:
Consider prefixing with a `.` so these are seen as "private" and don't show up in the help.

Reply:
As far as I can tell, they don't show up in the help unless you add a `## ` comment at the end of the line.

FWIW it seems that a `--` prefix is the [right way to render make targets private](https://stackoverflow.com/questions/26063839/how-can-i-make-a-target-private-in-gnu-make-for-internal-use-only-or-how-to).